packages/@aws-cdk/aws-eks/test/integ.eks-cluster.cn.ts

```diff
@@ -0,0 +1,27 @@
+import { Vpc } from '@aws-cdk/aws-ec2';
```

Comment:
I think it’s ok for this to just be tested through a unit test instead of an integ test. I am actually working on consolidating all the integ tests here into a couple of tests because it really hurts agility.

Reply:
OK I will remove this integ test.